templates/pkg/resource/sdk_find_read_many.go.tpl

```diff
@@ -51,6 +58,19 @@ func (rm *resourceManager) sdkFind(
 	return &resource{ko}, nil
 }
 
+// requiredFieldsMissingFromReadManyInput returns true if there are any fields
+// for the ReadMany Input shape that are required but not present in the
+// resource's Spec or Status
+func (rm *resourceManager) requiredFieldsMissingFromReadManyInput(
+	r *resource,
+) bool {
+{{- if $customCheckMethod := .CRD.GetCustomCheckRequiredFieldsMissingMethod .CRD.Ops.ReadMany }}
+return rm.{{ $customCheckMethod }}(r)
```

Comment:
Needs one TAB indent above.

Reply:
To confirm: tab `return rm.{{ $customCheckMethod }}(r)` on line 68?

Reply:
yuppers.